draft-ietf-quic-http.md

```diff
@@ -1664,9 +1664,10 @@ HTTP_UNEXPECTED_FRAME (0x0013):
 HTTP_REQUEST_REJECTED (0x0014):
 : A server rejected a request without performing any application processing.
 
-HTTP_GENERAL_PROTOCOL_ERROR (0x00FF):
-: Peer violated protocol requirements in a way which doesn't match a more
-  specific error code, or endpoint declines to use the more specific error code.
+HTTP_SETTINGS_ERROR (0x00FF):
+: An endpoint detected an error with a SETTINGS frame: a duplicate setting was
```

Comment:
Maybe "detected an error _in the payload of_ a SETTINGS frame"?

I suggest this assuming that HTTP_MALFORMED_FRAME is the error code to be used when seeing a badly-formatted SETTINGS frame (for example a SETTINGS frame carrying one byte of data: 0xff).

Reply:
I added you suggestion for payload. 

The ultimate goal is to replace MALFORMED_FRAME with FRAME_SIZE_ERROR. That should cover the case you mention. At that point in time we an add some text to suggest which type of error code should be used.